polaris-core/src/main/java/org/apache/polaris/core/entity/PolarisPrivilege.java

```diff
@@ -102,7 +102,9 @@ public enum PolarisPrivilege {
   CATALOG_ROLE_FULL_METADATA(67, PolarisEntityType.CATALOG_ROLE),
   CATALOG_ROLE_MANAGE_GRANTS_ON_SECURABLE(68, PolarisEntityType.CATALOG_ROLE),
   CATALOG_ROLE_MANAGE_GRANTS_FOR_GRANTEE(69, PolarisEntityType.CATALOG_ROLE),
-  ;
```

Comment:
nit: keeping this semicolon on a new line allows for pure "add" diffs in PRs and simplifies linking enum values to commits that truly added them :)